dietpi/dietpi-cloudshell

```diff
@@ -1054,11 +1057,11 @@
 
 	}
 
-	# Tor Relay: Start monitoring if not already active
+	# Tor Relay: Start displaying if not already
 	NYX_ACTIVE=
-	Update_Scene_9(){ [[ $NYX_ACTIVE ]] || { clear; nyx & NYX_ACTIVE=1; }; }
+	Update_Scene_9(){ [[ $NYX_ACTIVE ]] || { clear; fg & NYX_ACTIVE=1; }; }
```

Comment:
Just in case there is any other job sent to background, the wrong one would be pulled to foreground 🤔.